### ISSUE TITLE
Non-serializable Python objects cannot be set as Global variables and passed between pre/post execution scripts and test cases.

### ISSUE THREAD

**Issue by CJHoffmann** [Luxoft/Twister#8]
As stated in the user guide, , serializable Python objects, like a telnet object, can be created in one TC, saved as a global variable using setGlobal(), and can then be used in other TCs in the test script using getGlobal(). However, this function only seems to work within test cases.

The set/getGlobal() functions do not work in a pre/post execution script.  I understand that xmlrcplib can be imported, a connection to the CE can be made, and the getGlobalVariable() function can then be used instead.  However, this also does not pass serializable Python objects. Other object types, strings, numbers, bools, ... can be set as globals and passed to TCs.

**Comment by croqaz** [comment]
The only global variables available in ALL Execution Processes and in pre/ post execution scripts are the variables you can see in the Java applet, in Configuration/ Global Parameters. All those variables are strings and are serializable.
When you setGlobal in a test and save a Python object, that object is remembered inside the Execution Process that is running the respective test. The object can be used by any test that runs on that specific Execution Process.
So in conclusion, your pre/ post scripts cannot access Python objects saved inside one specific EP.

**Comment by bogdanpopescu** [comment]
@CJHoffmann  do you agree to close the issue ?

**Comment by CJHoffmann** [comment]
For now we have a work-around by doing the initialization in the first test case of a test suite.  So, for now, this issue can be closed.
